libid/fractals.cpp

```diff
@@ -728,3 +881,29 @@ int other_julia_fp_per_pixel()
     }
     return 0;
 }
+
+int burning_ship_fp_fractal()
+{
+    DComplex z, q;
+    double real_imag;
+    int degree = (int) g_params[2];
```

Comment:
Declare `const`

Reply:
I don't understand this one. It causes peoblems when I try to add it. (now in burning_ship.cpp)
Not done.